### PR TITLE
Disabling rmc in KVM script

### PR DESCRIPTION
**Pull request by satwinder0867**
@sudeeshjohn

### CODE REVIEW

**Comment by satwinder0867** [comment]
These changes are made as the RMC container is not reqd for the Libvert cluster as per discussion b/w Archana and Manju.
^ ^@mkumatag

File: modules/5_install/templates/install_vars.yaml [hunk 1]
```diff
@@ -18,3 +18,4 @@ chronyconfig:
       options: ${lookup(item, "options", "iburst")}
 %{ endfor ~}
 %{ endif ~}
+powervm_rmc: false
```
Comment (mkumatag) [inline modules/5_install/templates/install_vars.yaml]: LGTM

Just add a new line at the end

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by mkumatag** [comment]
Let's get the review comments from @yussufsh and @Prajyot-Parab as well.

**Comment by satwinder0867** [comment]
@yussufsh @Prajyot-Parab 
Extra line is added at the end, plz merge it

**Comment by mkumatag** [comment]
@satwinder0867 fix the commit author, it says root at the moment,

how to fix - https://stackoverflow.com/questions/25339671/git-uses-root-instead-of-my-username-while-commit-linux

**Comment by mkumatag** [comment]
If you look at the commit is still says - `root authored and satwinder0867 committed 2 hours ago` seems like still stuck with co-authored,

**Comment by satwinder0867** [comment]
@mkumatag @yussufsh 
i made changes again, hope this time its good to merge
let me know if any more changes are reqd.

**Comment by Prajyot-Parab** [comment]
@satwinder0867 Please refer to the below format, try to sign off your commit properly before pushing `git commit -m <message> -s`

```
<Commit Message>
Signed-off-by: <GH username> <Mail>
```

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

**Review by mkumatag — APPROVED** [review]
LGTM

**Review by yussufsh — APPROVED** [review]
lgtm

**Pull request merged**